addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -652,6 +683,90 @@ QUnit.test('composer text input cleared on message post', async function (assert
     );
 });
 
+QUnit.test('composer inputs cleared on message post in extended composer', async function (assert) {
+    assert.expect(10);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: 'channel',
+                id: 20,
+                mass_mailing: true,
+                name: "General",
+            }],
+        },
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.method === 'message_post') {
+                assert.step('message_post');
+                assert.ok(
+                    'body' in args.kwargs,
+                    "body should be posted with the message"
+                );
+                assert.strictEqual(
+                    args.kwargs.body,
+                    "test message",
+                    "posted body should be the correct one"
+                );
+                assert.ok(
+                    'subject' in args.kwargs,
+                    "subject should be posted with the message"
+                );
+                assert.strictEqual(
+                    args.kwargs.subject,
+                    "test subject",
+                    "posted subject should be the correct one"
```

Comment:
"... should be the correct one" is not great. If it fails, we have to read the code and the test as a whole to understand what is implied by "correct one". I would reword with "... should be the one typed in <subject/text> input"